toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -12,10 +12,14 @@ import (
 
 // SystemConfig defines how each system present on the image is supposed to be configured.
 type SystemConfig struct {
-	Hostname             string                    `yaml:"Hostname"`
-	AdditionalFiles      map[string]FileConfigList `yaml:"AdditionalFiles"`
-	PostInstallScripts   []Script                  `yaml:"PostInstallScripts"`
-	FinalizeImageScripts []Script                  `yaml:"FinalizeImageScripts"`
+	Hostname              string                    `yaml:"Hostname"`
+	PackagesToAddLists    []string                  `yaml:"PackagesToAddLists"`
```

Comment:
Should that be PackageListsToAdd instead?

Reply:
Probably not. It is not "package lists" that are being added to the image but rather "packages".

Reply:
Hmm. Daniel was also confused by the name. So, these will probably need to change.

Reply:
Change the names.